### ISSUE TITLE
Spacing issue in cls file

### ISSUE THREAD

**Issue by rikudoukarthik** [posquit0/Awesome-CV#322]
I recently faced an issue when trying to make a CV using the AwesomeCV template. My problem was that it considered regular content and R code content as two different kinds of content, and gave different spacing for each. 

When I looked into the `cls`, I noticed that in addition to `\acvSectionContentTopSkip` and `\acvSectionTopSkip` values, there was another `\vspace{-3mm}` in the `%Define a subsection for CV` and `%Define a paragraph for CV` options. I chose to remove the -3mm from here, and also changed `\acvSectionContentTopSkip` from 2.5mm to 0.5mm. Now things seem much better.

Could you maybe explain why exactly there is `\vspace{\acvSectionContentTopSkip}` as well as `\vspace{-3mm}` in the original code?

https://github.com/posquit0/Awesome-CV/blob/4be6c213c8631a1fd68d1ad452e88b722b911f78/awesome-cv.cls#L561-L577

**Comment by jakec-dev** [comment]
Yep, I have the same issue. My third cventry is squished too closely to my second entry. The fix provided by @rikudoukarthik worked for me. What are the consequences for this fix?
